draft-ietf-quic-manageability.md

```diff
@@ -676,18 +676,18 @@ secret is retained by the observer.
 
 ## Flow Association {#sec-flow-association}
 
-The QUIC connection ID (see {{rebinding}}) is designed to allow an on-path
-device such as a load-balancer to associate two flows as identified by
-five-tuple when the address and port of one of the endpoints changes; e.g. due
-to NAT rebinding or address migration. An observer keeping flow state
-can associate a connection ID, if present, with a given flow, and can associate
-a known flow with a new flow when when observing a packet sharing the same
-connection ID in the same direction between client and server and sharing one
-endpoint address (IP address and port) with the known flow.
-
-However, since the connection ID may change multiple times during the lifetime
-of a flow, and the negotiation of connection ID changes is encrypted, packets
-with the same 5-tuple but different connection IDs might or might not belong to
+The QUIC connection ID (see {{rebinding}}) is designed to allow a coordinating
+on-path device, such as a load-balancer, to associate two flows when one of the
+endpoints changes address or port.  This change can be due to NAT rebinding or
+address migration.
+
+The connection ID must change upon intentional address change by an endpoint,
+so it is only possible for a passive observer to link unintended changes of
+address using the connection ID.
+
+Since the connection ID may change multiple times during the lifetime
+of a connection, and connection ID negotiation is encrypted, packets with the
+same five-tuple but different connection IDs might or might not belong to
 the same connection.
```

Comment:
```suggestion
A network function that uses the connection ID to associate flows must
be robust to the failure of this technique. Since the connection ID may
change multiple times during the lifetime of a connection, and 
connection ID negotiation is encrypted, packets with the
same five-tuple but different connection IDs might or might not belong to
the same connection. Likewise, packets with the same connection ID but
different five-tuples might not belong to the same connection, either.
```

Reply:
@ianswett editing by editors led to this change (covering unintentional address change and the fact that observation techniques are brittle); wdyt?

Reply:
I'd prefer this paragraph without the first sentence, since that's reintroducing text suggesting a middlebox might do this.  If we are going to keep it, I think it needs to be reworded slightly.

The last sentence is a nice addition, though.